### PR TITLE
Created PhysicalClass, Hunter and Warrior Classes

### PR DESCRIPTION
**Pull request by AlejandroWoloch**
There might be some things missing due to not havig the super and releated classes finished yet

### CODE REVIEW

File: src/Hunter.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		}
+	}
+	
+	@Override
+	public void equipWeapon(Weapon weapon) {
+		if(weapon != null && weapon instanceof Bow) {
+			this.equipedWeapon = weapon;
+		}
+	}
+
```
Comment (rightonhana) [inline src/Hunter.java:39]: Remember to override toString and equals methods

File: src/PhysicalClass.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		for (PhysicalSkill skill : this.getSkillList()) {
+			if(skill.getId() == id) {
+				toRemove = skill;
+			}
+		}
+		
+		if(toRemove != null) {
+			this.skillList.remove(toRemove);
+		}
+	}
```
Comment (rightonhana) [inline src/PhysicalClass.java:57]: Remember to override toString and equals methods

File: src/Warrior.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+		}
+	}
+	
+	@Override
+	public void equipWeapon(Weapon weapon) {
+		if(weapon != null && weapon instanceof Sword) {
+			this.equipedWeapon = weapon;
+		}
+	}
+
```
Comment (rightonhana) [inline src/Warrior.java:40]: Remember to override toString and equals methods

File: src/PhysicalClass.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
+
+public abstract class PhysicalClass extends PlayableCharacter{
+	
+	//Atributes
+	private static final int maxStamina = 100;
+	private int stamina;
+	protected ArrayList<PhysicalSkill> skillList;
+	private int maxSkills
```
Comment (ducampcarlos) [inline src/PhysicalClass.java]: Missing ';'

File: src/PhysicalClass.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+	public PhysicalClass(String name) { //For brand new PhysicalCharacters
+		super(name);
+		this.setStamina(maxStamina);
+	}
+	
+	public PhysicalClass(byte level, String name, int experience, Weapon weapon, Armor armor, int stamina) {  //Used to instance with information from JSON
+		super(level,name,experience,weapon,armor);
+		this.setStamina(stamina);
+	}
+
```
Comment (ducampcarlos) [inline src/PhysicalClass.java:24]: We're not going to use items (for now)

File: src/Hunter.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+	
+	//Constructors
+	public Hunter(String name) {	//For brand new Hunters
+		super(name);
+	}
+	
+	public Hunter(byte level, String name, int experience, Weapon weapon, Armor armor, int stamina) {  //Used to instance with information from JSON
+		super(level,name,experience,weapon,armor);
+		this.setStamina(stamina);
+	}
```
Comment (ducampcarlos) [inline src/Hunter.java:14]: We're not going to use items (for now)

File: src/Warrior.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+	
+	//Constructors
+	public Warrior(String name) {	//For brand new Hunters
+		super(name);
+	}
+	
+	public Warrior(byte level, String name, int experience, Weapon weapon, Armor armor, int stamina) {  //Used to instance with information from JSON
+		super(level,name,experience,weapon,armor);
+		this.setStamina(stamina);
+	}
```
Comment (ducampcarlos) [inline src/Warrior.java:14]: We're not going to use items (for now)

File: src/PhysicalClass.java [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+		PhysicalSkill toRemove = null;
+		
+		for (PhysicalSkill skill : this.getSkillList()) {
+			if(skill.getId() == id) {
+				toRemove = skill;
+			}
+		}
+		
+		if(toRemove != null) {
+			this.skillList.remove(toRemove);
```
Comment (rightonhana) [inline src/PhysicalClass.java]: Verify if the Arraylist is empty

**Comment by AlejandroWoloch** [comment]
Okey, I commented code releated to iteams and also to skills until we merge the Ability Class, Added toString and equals. I still have to add some exceptions, but can we discuss how to do that in tomorrow's call?

**Comment by ducampcarlos** [comment]
Everything looks okay
I think we can merge it and later add any changes needed

**Comment by AlejandroWoloch** [comment]
You have my Woloch's seal of approval

**Pull request merged**